prow/config/templates/ing-Ingress.yaml

```diff
@@ -45,4 +38,11 @@ spec:
             name: pushgateway-external
             port:
               number: 80
+      - path: /*
+        pathType: ImplementationSpecific
+        backend:
+          service:
+            name: deck
+            port:
+              number: 80
```

Comment:
Order matters? i guess you're configuration all `/*` **except** `/metrics` and` /hooks` ?

Reply:
I think so. I think it evaluates the path in the order they're specified. Weirdly the upstream prow still has `/*` so I don't really understand how it's working for them.

Reply:
Actually I think upstream prow has `/`, not `/*`:

https://github.com/kubernetes/test-infra/blob/cfaeafa474e934d3ed0454e1bb08edb55fcebb62/config/prow/cluster/starter/starter-s3.yaml#L491

Reply:
Oh interesting, I took it from their Prow canary test setup - https://github.com/kubernetes/test-infra/blob/cfaeafa474e934d3ed0454e1bb08edb55fcebb62/config/prow/cluster/ing_ingress.yaml

Reply:
Going with the spec as defined by the canaries, since these have been validated as working already.

Reply:
We discussed this offline and agreed that we don't actually know which of these is best practice.  

We're going w/ the canary test setup as that one is tested.